modules/ROOT/pages/database-internals/transaction-management.adoc

```diff
@@ -54,7 +54,10 @@ db.transaction.timeout=10s
 ====
 
 Configuring transaction timeout does not affect transactions executed with custom timeouts (e.g., via the Java API or Neo4j Drivers), as the custom timeout overrides the value set for `db.transaction.timeout`.
-Note that the timeout value can only be overridden to a value that is smaller than that configured by `db.transaction.timeout`.
+Note that the timeout value can be overridden to a value smaller than that configured by `db.transaction.timeout`.
```

Comment:
Without the only this sentence dons't make much sense to me. Reading up to here I'd think "Of course it can! If you don't tell me otherwise, I'll assume it can be overridden to anything".

It also leaves me wondering "okay, and what about values greater than `db.transaction.timeout`?"

Reply:
ok, I see. Put the word 'only' back into the sentence. 
I have another question: should we clarify here that it's true only in versions 5.1 and 5.2?

Reply:
I think it's clear enough as well.

Furthermore, 4.4 behaves just like 5.1 on this regard and everything before 4.4 is EOL, so I'd argue it's not a big deal if the documentation here doesn't apply to e.g., 4.2.